core/arch/instrlist.c

```diff
@@ -245,7 +245,8 @@ check_translation(instrlist_t *ilist, instr_t *inst)
 #if defined(CLIENT_INTERFACE) && defined(ARM)
     if (instr_is_meta(inst)) {
         dr_pred_type_t auto_pred = ilist->auto_pred;
-        if (auto_pred != DR_PRED_AL && auto_pred != DR_PRED_NONE) {
+        if (auto_pred != DR_PRED_AL && auto_pred != DR_PRED_OP &&
```

Comment:
Actually never mind they're almost done anyway, I thought there'd be a delay on the appveyor queue due to the prior commit.